src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.38.0'
+__version__ = '1.38.1'
```

Comment:
Think this should be a minor b/c adding new `archive_root` and `restore_root` methods.

Reply:
That's a fair point.